src/outputs/outputs_main.py

```diff
@@ -142,6 +147,19 @@ def run_outputs(
         )
         OutputMainLogger.info("Finished GB SAS output.")
 
+    # Running NI SAS output
+    if config["global"]["output_ni_sas"]:
+        OutputMainLogger.info("Starting NI SAS output...")
+        output_ni_sas(
+            ni_full_responses,
+            config,
+            write_csv,
+            run_id,
+            ultfoc_mapper,
```

Comment:
We have a column "foc". We don't need a mapper. Just rename "foc" into "f_owner" and make sure the column appear in correct order.